### ISSUE TITLE
Failed to migrate manifest

### ISSUE THREAD

**Issue by Hrxn** [dyc3/steamguard-cli#283]
### Version

steamguard-cli 0.10.1

### What happened?

steamguard correctly recognizes the `maFiles` directory from SDA, but apparently fails at parsing somehow?

I've refreshed the login in SDA (latest version `v1.0.14`) and turned off the encryption. SDA is working normally so far..

### Steps To Reproduce

OS: Windows 10 22H2 (10.0.19045.3155) (i.e. the latest stable Windows 10, basically)
Shell: PowerShell 7.3.5

### Relevant log output

```Shell
PS C:\Users\Hrxn> steamguard -m D:\Portable\SDA\maFiles -v debug
DEBUG - Args { global: GlobalArgs { username: None, all: false, mafiles_path: Some("D:\\Portable\\SDA\\maFiles"), passkey: None, verbosity: Debug, no_update_check: false, http_proxy: None, proxy_credentials: None, danger_accept_invalid_certs: false }, sub: None, code: CodeCommand { offline: false } }
INFO - reading manifest from D:\Portable\SDA\maFiles
DEBUG - loading manifest: "D:\\Portable\\SDA\\maFiles\\manifest.json"
INFO - Migrating manifest
DEBUG - deserializing manifest: version None
DEBUG - loading all accounts for migration
DEBUG - loading entry: "D:\\Portable\\SDA\\maFiles\\<redacted>.maFile"
ERROR - Failed to migrate manifest: An unexpected error occurred during manifest migration: Failed to load some accounts: [DeserializationFailed(Error { path: Path { segments: [Map { key: "Session" }, Map { key: "SessionID" }] }, original: Error("invalid type: null, expected a string", line: 1, column: 1573) })]
ERROR - An unexpected error occurred during manifest migration: Failed to load some accounts: [DeserializationFailed(Error { path: Path { segments: [Map { key: "Session" }, Map { key: "SessionID" }] }, original: Error("invalid type: null, expected a string", line: 1, column: 1573) })]

Caused by:
    Failed to load some accounts: [DeserializationFailed(Error { path: Path { segments: [Map { key: "Session" }, Map { key: "SessionID" }] }, original: Error("invalid type: null, expected a string", line: 1, column: 1573) })]
DEBUG - Checking for updates to dyc3/steamguard-cli v0.10.1
DEBUG - No update available
PS C:\Users\Hrxn>
```

**Comment by dyc3** [comment]
This is definitely a bug, thanks for posting here. The migration stuff needs to be a little more fault tolerant.

For this specific case, if you know how how to edit JSON, make the `SessionID` field into an empty string `""`, eg. `"SessionID": ""`. Or alternatively make the entire `Session` object `null`, eg. `"Session": null`, because we throw that field out during the migration.

**Comment by dyc3** [comment]
this should be fixed in `v0.10.2`

**Comment by Hrxn** [comment]
Yeah, apparently, seems to be working fine now!
Thanks for the quick fix, cheers 👍
